argoverse/data_loading/argoverse_forecasting_loader.py

```diff
@@ -77,7 +77,7 @@ def seq_df(self) -> pd.DataFrame:
         return _read_csv(self.current_seq)
 
     @property
-    def agent_traj(self) -> np.ndarray:
+    def agent_traj(self) -> Any:
```

Comment:
maybe we can cast the return arg to np.array() since, np.column_stack() should hopefully only return an array